Cargo.toml

```diff
@@ -6,6 +6,9 @@ members = [
     "hacspec",
     "secret-integers",
     "abstract-integers",
+    "hacspec-examples/hacspec-chacha20",
```

Comment:
We should update the readme to say what's the difference between `spec-examples` and `hacspec-examples`. And then add a readme to `hacspec-examples` that explains the structure and how to type check and translate them.

Reply:
Yes, can do!

Reply:
Done !